app/models/user.py

```diff
@@ -204,8 +204,8 @@ def generate_fake(count=100, **kwargs):
                 confirmed=True,
                 role=role,
                 **kwargs)
-            if u.is_applicant():
-                u.applicant_profile = ApplicantProfile.generate_fake()
+            if u.role.id == 1:
```

Comment:
can we case on the enum val itself? instead of ==1?

Reply:
👍 good catch